### ISSUE TITLE
CXXABI_1.3.9 not found error on v1.3.0

### ISSUE THREAD

**Issue by bencripps** [blueconic/node-oom-heapdump#13]
Recently we upgraded to 1.3.0 from 1.2.1 and we now see this error when our application starts up:

```
        ^Error: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by node-oom-heapdump/build/Release/node_oom_heapdump_native.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:807:18)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

Downgraded to 1.2.1 resolves the issue for us -- is this a breaking change, or is this something we need to change about our environment?

**Comment by paulrutter** [comment]
1.3.0 was about adding pre-built binaries, nothing more. Judging from the error, it was built on a newer gcc version than your environment has.

Three options:
* use previous version, which has exactly the same features
* update gcc in your environment
* use the fallback-to-build command

Can you check if this resolves your issue?

**Comment by bencripps** [comment]
Thanks for the prompt reply @paulrutter!

We're going to pin to the working version, but I have a few questions:

1. I'm not sure how to use the fallback-to-build command? Do you have any insights?
2. The fact that we need to change the code/behaivor of our app seems to indicate that this is a breaking change -- does it make sense to revert this and publish a new patch, and then publish a new major which consumes the change?

**Comment by paulrutter** [comment]
1. Where you usually call npm install, now call npm install --build-from-source. This will skip the pre-built binaries and build from source
2. I hadn't anticipated that this would be a breaking change, since i didn't think if compiler mismatches. I'll think about dropping this version and republish as 2.x.

**Comment by paulrutter** [comment]
I've published a new 1.3.1 version, which is without prebuilt binaries.
The 2.0.0 version contains the prebuilt binaries as might not be backwards compatible, depending on the environment in which the binaries are used.

**Comment by bencripps** [comment]
Great, thank you!
